package.json

```diff
@@ -92,11 +92,12 @@
     "@babel/preset-env": "^7.14.4",
     "@babel/preset-flow": "^7.14.5",
     "@babel/preset-react": "^7.14.5",
+    "@testing-library/dom": "^8.1.0",
```

Comment:
This makes it easier to control the version of the underlying dom library; not strictly needed in the current version of the PR but this is nice to have IMO. Otherwise we'll keep the initial version until `@testing-library/react` upgrades its minimum.

Reply:
Cool, it looks harmless to me.